.github/workflows/actions.yml

```diff
@@ -50,6 +50,8 @@ jobs:
           - tx16s
           - nv14
           - el18
+          - pl18
+          - pl18ev
```

Comment:
Do we really need to run tests for pl18ev as well as the pl18?

Reply:
Well, I just observe the fact that nv14 and el18 are there and make the decision.

Reply:
True... that changed once the AFHDS handling separated so they have some completely different code paths. I was just thinking the PL18EV might contain everything in the PL18, so just that one could be built rather than both. After all, we're not doing unit tests for ALL radios, just enough that are significantly different (IMO there are a couple others that could also), since firmware will be built for all of them.